i18n-helpers/src/directives.rs

```diff
@@ -0,0 +1,130 @@
+use regex::{escape, Regex};
+
+#[derive(Debug, PartialEq)]
+pub enum Directive {
+    Skip,
+    TranslatorComment(String),
+}
+
+pub struct DirectiveFinder {
+    pattern_re: Regex,
```

Comment:
I would suggest either `pattern` or just `re` — the current name seems a bit long and it's almost like you're encoding the type (`Regex`) into it (`_re`).

Reply:
Acknowledged; I will simplify this and just call it re.